internal/logging/logging.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/statechannels/go-nitro/types"
 )
 
+const TraceLogLevel = -5
```

Comment:
Nitpicks:
1. We could rename this to `LevelTrace` to follow the naming pattern of the [existing levels](https://pkg.go.dev/log/slog#Level)
2. We could define this as `slog.Level` const instead of `int`
```suggestion
const LevelTrace slog.Level = -5
```

Reply:
Very good point. Done in cda2af8a.